db/migrate/20191010195747_remove_portfolio_item_disploy_name.rb

```diff
@@ -0,0 +1,5 @@
+class RemovePortfolioItemDisployName < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :portfolio_items, :display_name, :string
```

Comment:
Would it make sense to remove the `name` column and then rename `display_name` to `name` to preserve the Products names users see today?